src/scaling.jl

```diff
@@ -0,0 +1,192 @@
+"""
+    Scaling <: Transform
+
+Linearly scale the data as `ax + b`, according to some statistics `a` and `b`.
+"""
+abstract type Scaling <: Transform end
+
+"""
+    IdentityScaling
+
+Represents the no-op scaling which simply returns the data it is applied on.
+"""
+struct IdentityScaling <: Scaling end
+# Convenience method
+IdentityScaling(args...; kwargs...) = IdentityScaling()
+
+apply!(A::AbstractArray{T}, scaling::IdentityScaling; kwargs...) where T <: Real = A
```

Comment:
```suggestion
apply!(A, ::IdentityScaling; kwargs...) = A
```

Reply:
This caused method ambiguity with `apply!(::AbstractArray, ::Transform; kwargs...)`. I changed this to `_apply!`, which solved the ambiguity but means it goes through the outer `apply` method wastefully.

Reply:
`but means it goes through the outer apply method wastefully.` 

Doesn't the outer method enable the support for dims and inds so that it doesn't need to be re-implemented for your transform?